src/gainmap.c

```diff
@@ -658,6 +655,21 @@ avifResult avifRGBImageComputeGainMap(const avifRGBImage * baseRgbImage,
         }
     }
 
+    // Compute the headroom for the base and alterate images. The gainmap is the log-ratio of the
+    // higher-headroom (more HDR) image to the lower-headroom (less HDR) image. Reverse the gainmap
+    // if this is the case.
```

Comment:
I assume it is this change that fixed the test failures on Windows. Do you know why the tests only fail on Windows?

Seel also my comment (marked with IMPORTANT) in tests/gtest/avifgainmaptest.cc. I wonder if that is why the tests failed on Windows. Still, it is strange that the tests did not fail on all platforms.

Reply:
Yes, this is why the test failed. I just assumed that only AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP was defined on the Windows machine. I developed the patch locally on Linux, and the test failed locally for me (which would be consistent with only AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP being defined on the Windows bot).